modules/ROOT/pages/restarting-a-node.adoc

```diff
@@ -1,78 +1,118 @@
-= Stop, Restart, or Update an Anypoint Private Cloud Edition Node
+= Update Nodes
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-This topic describes the processes for removing an Anypoint Platform PCE node from active service, and returning it to service when ready.
+== Before You Begin
 
-== Notes
+* Set up and test the backup procedure on all clusters.
+* Perform a backup before attempting node maintenance to preserve data in the event of a failure.
 
-[NOTE]
-To preserve your data in case of system failure, ensure that your system is backed up before attempting node maintenance.
-[NOTE]
-The steps below must be performed on a single Anypoint Platform PCE node at a time.  If maintenance is required on more than one node, the entire process must be completed on a single node before the next node is addressed.
+== Cluster and Application Updates
 
-== Drain the Node
+[IMPORTANT]
+The following steps must be performed on one Anypoint Platform Private Cloud Edition node at a time to minimize or eliminate component breakage during the update process. The entire process must be completed on each node before the next node is addressed. Verify the health status of all cluster components and applications after each node update before proceeding with the next node.
 
-Before any maintenance or restart of a node can be performed, it must be removed from the cluster and its workload reassigned to the other nodes in the cluster.
+=== Verify Cluster Health Status
 
-. Check if the firewalld service is running and enabled:
+Verify that the cluster is healthy before starting the process. Otherwise, many of the assumptions made in this process are not valid.
```

Comment:
More to the point:

"If your cluster is not healthy before you begin the upgrade, the upgrade fails. You test the health of your cluster by verifying certain command output and verifying that all pods are running."